### ISSUE TITLE
[Feature Request] Spotlight (tap-to-target) guide tour

### ISSUE THREAD

**Issue by meyt** [vuetifyjs/vuetify#5987]
### Problem to solve
Guide guest users about page elements by cover the whole page and focus target area with some description. user can go to next tip or skip the tour.

### Proposed solution
Examples:
[![alt](https://raw.githubusercontent.com/KeepSafe/TapTargetView/master/.github/video.gif)
](https://sjwall.github.io/MaterialTapTargetPrompt/examples.html)
https://github.com/KeepSafe/TapTargetView

[material.io documentation ](https://material.io/design/communication/onboarding.html#quickstart-model)

**Comment by sh7dm** [comment]
Good idea! I think that it can be done with new mixin. Maybe I'll work on it.

**Comment by LordArma** [comment]
Very good idea 👍👍👍

**Comment by MajesticPotatoe** [comment]
There was a PR at one point but was closed due to inactivity/merge issues. Plans to bring this in with 2.0.

**Comment by aggarwaldev** [comment]
Duplicate of https://github.com/vuetifyjs/vuetify/issues/1171
